package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This visualization plugin is like a table, but with computed columns",
   "main": "index.js",
   "kibana": {
-    "version": "5.5.0"
+    "version": "kibana"
```

Comment:
This will not work. The special `kibana` version can only be used from `core_plugins` (inside the Kibana source code), to be able to always run. The plugin installer would not allow this to be installed anywhere.

Reply:
It does work in dev mode :). Indeed to be set to 6.1.0 or 6.1.1 if in production or build for release. Thanks again for support!